app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/ContributorsActivity.java

```diff
@@ -51,8 +54,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case R.id.item_repository:
-                Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse("https://github.com/DroidKaigi/conference-app-2017"));
-                startActivity(intent);
+                navigator.navigateToWebPage("https://github.com/DroidKaigi/conference-app-2017");
```

Comment:
I think View can call ViewModel's event handler like:
```java
viewModel.onClickRepositoryMenu(); // I am not a fun of this naming. Any ideas?
```
It's not perfect but not bad in MVVM way :v: